src/jobs/go-architect.yaml

```diff
@@ -0,0 +1,12 @@
+description: |
```

Comment:
Why do we need this?

Reply:
A description? It's a sort of documentation for people setting up new workflows. CircleCI orbs are public so it makes it look nicer here https://circleci.com/orbs/registry/?filterBy=all&query=giantswarm. And other jobs already have it so I was trying to be consistent.

Reply:
I meant this job :D

Reply:
Sorry that was quite a shortcut.

Reply:
Heh, okay. This approach means we don't need to download the architect binary on the CircleCI runner. We can run it in a container and get a known version of architect. It also means less to get out of sync between repos which currently need to implement this step individually in each CircleCI config. We can go ahead and switch all repos to use this orb and then migrating to go modules is easier.

Reply:
I understand what the job is doing. My question is what it is supposed to be used for?